docs/index.md

```diff
@@ -21,5 +21,8 @@ You can follow our getting started guide, which will provide instructions on how
 - [Webhooks](usage/webhooks.md)
   - [Register a Webhook](usage/webhooks.md#register-a-webhook)
   - [Process a Webhook](usage/webhooks.md#process-a-webhook)
+- [Create a `CustomSessionStorage` Solution](usage/customsessions.md)
+  - [Callback Methods](usage/customsessions.md#callback-methods)
+  - [Usage Example with Redis](usage/customsessions.md#example-usage)
```

Comment:
I'm not sure we need every subsection to figure here - I think the 'example with redis' one is valid because it helps surface it, but I don't think it's necessary for 'callback methods' to be here.

Reply:
That's fair! It just seemed like our convention was to link to level 2 headings.